src/test/java/io/openliberty/tools/intellij/it/MavenSingleModMPProjectTest.java

```diff
@@ -63,14 +63,16 @@ public class MavenSingleModMPProjectTest extends SingleModMPProjectTestCommon {
     private final String BUILD_FILE_OPEN_CMD = "Liberty: View pom.xml";
 
     /**
-     * The path to the integration test reports.
+     * The paths to the integration test reports. The first is used when maven-surefire-report-plugin 3.4 is used and the second when version 3.5 is used.
```

Comment:
should be `maven-failsafe-report-plugin` here

Reply:
I don't think there is a failsafe version, I believe surefire is used by both.